Bottom: c63967d0901f9bb6e4afd75305ddcf16d04d5569
Top:    f12ee2d321846fdc31b2c16758369fb83955a804
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-08-28 03:09:17 +0100

Refresh of add-virtual-trackball-code

---

diff --git a/src/Makefile.am b/src/Makefile.am
index 6333f45..6a0c335 100644
--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -161,7 +161,9 @@ LIBGTK_GDK_SRCS= \
 	hid/gtk/gtkhid-gdk.c
 
 LIBGTK_GL_SRCS= \
-	hid/gtk/gtkhid-gl.c
+	hid/gtk/gtkhid-gl.c \
+	hid/gtk/gui-trackball.c \
+	hid/gtk/gui-trackball.h
 
 GL_SRCS= \
 	hid/common/hidgl.c \
@@ -277,8 +279,6 @@ LIBGTK_SRCS = \
 	hid/gtk/gui-pinout-preview.h \
 	hid/gtk/gui-pinout-window.c \
 	hid/gtk/gui-top-window.c \
-	hid/gtk/gui-trackball.c \
-	hid/gtk/gui-trackball.h \
 	hid/gtk/gui-utils.c
 libgtk_a_SOURCES = ${LIBGTK_SRCS} hid/gtk/gtk_lists.h
